lib/changelog/prom_ex.ex

```diff
@@ -8,7 +8,9 @@ defmodule Changelog.PromEx do
       {PromEx.Plugins.Application, otp_app: :changelog},
       PromEx.Plugins.Beam,
       {PromEx.Plugins.Phoenix, router: ChangelogWeb.Router},
-      {PromEx.Plugins.Ecto, otp_app: :changelog, repos: [Changelog.Repo]}
+      {PromEx.Plugins.Ecto, otp_app: :changelog, repos: [Changelog.Repo]},
+      PromEx.Plugins.Oban
```

Comment:
Does this mean PromEx will run its stuff via Oban automatically now?

Reply:
That's correct! PromEx will automatically pick up on any Oban events that are emitted and those metrics will automagically find their way to the included Grafana Oban dashboard :)